stdlib/public/Platform/Platform.swift

```diff
@@ -311,8 +316,12 @@ public var SIG_DFL: sig_t? { return nil }
 public var SIG_IGN: sig_t { return unsafeBitCast(1, to: sig_t.self) }
 public var SIG_ERR: sig_t { return unsafeBitCast(-1, to: sig_t.self) }
 public var SIG_HOLD: sig_t { return unsafeBitCast(5, to: sig_t.self) }
-#elseif os(Linux) || os(FreeBSD)
+#elseif os(Linux) || os(FreeBSD) || os(Windows)
+#if os(Windows)
+public typealias sighandler_t = _sig_func_ptr
+#else
 public typealias sighandler_t = __sighandler_t
+#endif
 
 public var SIG_DFL: sighandler_t? { return nil }
```

Comment:
Yes, the values were same with Linux's one.  I'll change to add the comment about library and version.